stories/documentation/navigation/menu/menu-basic.stories.ts

```diff
@@ -30,30 +31,58 @@ export default {
 				type: 'boolean',
 			},
 		},
+		vertical: {
+			control: {
+				type: 'boolean',
+			},
+		},
 	},
 } as Meta;
 
 function getTemplate(args: MenuBasicStory): string {
 	const noBorder = args.noBorder ? `mod-noBorder` : '';
 	const header = args.header ? `mod-header` : '';
 	const s = args.s ? `mod-S` : '';
-	const disabled = args.disabled ? `disabled` : '';
-	return `
-	<nav class="menu ${s} ${noBorder} ${header}">
+	const vertical = args.vertical ? `mod-vertical` : '';
+	if (args.disabled)
+		return `
+	<nav class="menu ${s} ${noBorder} ${header} ${vertical}">
+		<ul class="menu-list">
+			<li class="menu-list-item">
+				<a href="#" class="menu-list-item-action" aria-current="page">
+					Menu 1
+				</a>
+			</li>
+			<li class="menu-list-item">
+				<a href="#" class="menu-list-item-action">
+					Menu 2
+				</a>
+			</li>
+			<li class="menu-list-item">
+				<span class="menu-list-item-action is-disabled">
+					Menu 3
+				</span>
```

Comment:
Par curiosité : Pourquoi pas uniquement virer le `href` ? Pour ne pas hériter de certains styles ?

Reply:
I'm afraid including anchor tags without the `href` attribute in the docs will be confusing for some users.

Reply:
This can be documented if required. Both work for me.